app/lib/common/themes/components/bottom_sheet_theme.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:acter/common/themes/colors/color_scheme.dart';
+import 'package:flutter/material.dart';
+
+var bottomSheetTheme = BottomSheetThemeData(
```

Comment:
Just a suggestion to ponder. I would really like to have these singleton themes be merged as one component theme file, makes it easier to change design stuff rather than looking over different files. But I would like to have general agreement on decision outcome!

Reply:
This can be subjective based on the different thought process but ya we can surely code this on either way. My main goal here is to have all the design standard should be implemented from Flutter `ThemeData` itself to make app UI consistent.